container/conductor/cli.py

```diff
@@ -76,8 +75,8 @@ def commandline():
     containers_config = decoding_fn(args.config)
     conductor_config = AnsibleContainerConductorConfig(containers_config)
 
-    logger.debug('Starting Ansible Container Conductor: %s', args.command)
-    logger.debug('Services: %s', conductor_config.services)
+    logger.debug('Starting Ansible Container Conductor: %s' % args.command,
+        services=conductor_config.services)
```

Comment:
This doesn't look right to me. The format string expects 1 argument, but we seem to be passing 2. Did we munge two `logger.debug` lines into something that's now broken?